src/iceflow/ingest/atm1b.py

```diff
@@ -384,8 +384,17 @@ class atm1bData(commonDataColumns):
     passive_footprint_synthesized_elevation: Series[pa.dtypes.Int32]
 
 
-@pa.check_types()
-def atm1b_data(filepath: Path) -> DataFrame[atm1bData]:
+class ATM1BData(IceFlowData):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        # Validate the data w/ pandera
+        # TODO: Does this result in pandera validating the common columns twice?
+        # The `super` call above would trigger the `IceFlowData`'s __init__,
+        # which include a call to `validate` on the common data columns.
```

Comment:
Another potential issue here. I read that `pandera` will try to cache validation, but would want to confirm if that's happening here. Maybe there's a better approach, where the schema is passed into the class at instantiation time?

Reply:
Worst case, could use some class state to avoid double-validation.